filament/include/filament/LightManager.h

```diff
@@ -236,22 +252,28 @@ class UTILS_PUBLIC LightManager : public FilamentAPI {
 
         /**
          * Whether screen-space contact shadows are used. This applies regardless of whether a
-         * Renderable is a shadow caster. This setting is currently only used for directional
-         * lights, ignored otherwise.
+         * Renderable is a shadow caster.
          * Screen-space contact shadows are typically useful in large scenes.
          * (off by default)
          */
         bool screenSpaceContactShadows = false;
 
         /**
-         * Number of ray-marching steps for screen-space contact shadows.
-         * (8 by default)
+         * Number of ray-marching steps for screen-space contact shadows (8 by default).
+         *
+         * CAUTION: this parameter is ignored for all lights except the directional/sun light,
```

Comment:
Do you think we should change that eventually?

Reply:
Yes I think so. That's why I didn't change the API. I mean if we had an editor, I could imagine the artist tweaking this per light ?

Reply:
Yeah I think we definitely want to tweak this per light.